### DISCUSSION TITLE
如何录制twitch的直播视频？

### DISCUSSION THREAD

**Discussion by 402699556** [go-olive/olive#49] [Q&A] [answered]
[[Shows]]
Platform = "twitch"
RoomID = "？？？？"
StreamerName = "Dakillzor"
录制时需要填写这种房间id，可是我发现tiwich和huya这些不一样，找不到房间号码如何进行填写呢？

**Comment by luxcgo** [comment, marked as answer]
可以参考模板文件中的配置，就是直播间网址的最后一个字符串。

**Reply by wwkk2580** [reply]
Parser这一项应该填什么？录制twitch时
